CBSE.py

```diff
@@ -197,6 +217,10 @@ def get_own_required_component_names(self):
         #TODO
         return ['r_' + c.get_type_name() for c in self.requiredComponents]
 
+    def get_own_further_dependencies(self):
```

Comment:
The own labels are for inheritance that's not implemented yet. Basically get_messages() would return all the messages handled by this component while get_own_messages() would return all the messages handled by this component that are not handled by the component's parent components.

It is very confusing and I'm thinking of dropping it to simplify the code, cleanup stuff, then reimplement inheritance from scratch.